common/lib/capa/capa/tests/test_files/js/xproblem.js

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
It is interesting that some of these files use two spaces for indentation and some use four. If these are ES2015 files then they should all use two spaces (ESLint will report any issues with this).